### ISSUE TITLE
solitary flower not an inflorescence

### ISSUE THREAD

**Issue by pcrooker** [Planteome/plant-ontology#712]
The inflorescence plant anatomical entry (PO:0009049) states it must have at least two flowers. What is a single flower reproductive shoot system called? I don't find any entry under:

   plant structure
        collective plant organ structure
             shoot system
                  reproductive shoot system

thanks.

**Comment by cooperl09** [comment]
You are correct that an inflorescence (PO:0009049) is defined as having 2 or more flowers.  A single flower (PO:0009046) is a determinate reproductive shoot system (PO:0025082).  
I am a bit confused why you had an issue finding it, it is a direct child of reproductive shoot system (PO:0025082).
![Screen Shot 2022-08-11 at 2 13 53 PM](https://user-images.githubusercontent.com/7378000/184243308-d21928ec-7a30-4dc0-848e-226542fe4fc4.png)

You could also get to it by typing "flower" into the search box on any page. 
https://browser.planteome.org/amigo/term/PO:0009046

**Comment by pcrooker** [comment]
I think I understand the misunderstanding - I'm referring to a single or solitary flower shoot system - flower, peduncle, bract(s), etc.

thanks.
